### PR TITLE
Validation changes

### PR DESCRIPTION
**Pull request by james-harper**
## Features
- Changed validation throughout to use Form Request Objects (140935067)

[Delivers [#140935067](https://www.pivotaltracker.com/story/show/140935067)]

### CODE REVIEW

**Comment by james-harper** [comment]
@EvanBarbour3 Ready for your review

File: app/Http/Requests/ApplicationRequestCancellationRequest.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+
+/**
+ * Class ApplicationRequestCancellationRequest
+ *
```
Comment (EvanBarbour3) [inline app/Http/Requests/ApplicationRequestCancellationRequest.php]: @author

File: app/Http/Requests/ApplicationRequestCancellationRequest.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * @package App\Http\Requests
+ */
+class ApplicationRequestCancellationRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
```
Comment (EvanBarbour3) [inline app/Http/Requests/ApplicationRequestCancellationRequest.php]: @author

File: app/Http/Requests/IpsStoreRequest.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: gregorykreko
+ * Date: 02/03/2017
+ * Time: 14:48
+ */
```
Comment (EvanBarbour3) [inline app/Http/Requests/IpsStoreRequest.php]: Remove this docblock

File: app/Http/Requests/RequestInitialisationRequest.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
+
+/**
+ * Class RequestInitialisationRequest
+ *
+ * @package App\Http\Requests
+ */
+class RequestInitialisationRequest extends Request
```
Comment (EvanBarbour3) [inline app/Http/Requests/RequestInitialisationRequest.php]: Maybe just `InitialisationRequest`

**Comment by james-harper** [comment]
@EvanBarbour3 

DocBlocks have been updated to include author, and class names have been changed to follow the same convention

File: app/Http/Requests/AccountUpdateRequest.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+/**
+ * Class AccountUpdateRequest
+ *
+ * @author JH
+ * @package App\Http\Requests
+ */
+class AccountUpdateRequest extends Request
+{
+    /**
+     * @author JH
```
Comment (EvanBarbour3) [inline app/Http/Requests/AccountUpdateRequest.php]: Tag should be below comment

File: app/Http/Requests/AccountUpdateRequest.php [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Http\Requests\Request;
```
Comment (EvanBarbour3) [inline app/Http/Requests/AccountUpdateRequest.php]: Request is unused

**Comment by james-harper** [comment]
@EvanBarbour3 I have removed the unused use statements and fixed the docBlock that was in the wrong place

**Pull request merged**